src/datepicker/datepicker-month-view.spec.ts

```diff
@@ -1,5 +1,6 @@
 import {TestBed, ComponentFixture} from '@angular/core/testing';
 import {createGenericTestComponent, isBrowser} from '../test/common';
+import {By} from '@angular/platform-browser';
```

Comment:
Is there a convention on which import block this one goes with? Or are the line breaks here arbitrary?